static/src/js/actions/project.js

```diff
@@ -311,6 +311,7 @@ export const getProjectCollections = () => async (dispatch, getState) => {
               browseFlag
               conceptId
               onlineAccessFlag
+              links
```

Comment:
Make sure that this change isn't also needed in the focusedCollection graphql call. I think sometimes it is unclear which graphql call is populating the store before the project page is displayed.

I think the test would be start logged in to EDSC with no collections in your project. Click on the collection to set it as the focused collection, which would call graphql in the focusedCollection action. Then click Download to go to the project page, and continue to download the collection. Then make sure everything was recorded correctly for the metrics

Reply:
Apologies I thought I'd responded here. After following these steps and a trying some more edge cases I was finding that I was able to add the collection recording correctly